bitnami/harbor/templates/core/core-svc.yaml

```diff
@@ -11,12 +12,12 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 spec:
-  {{- if  (eq .Values.ingress.controller "gce") }}
+  {{- if  (eq .Values.ingress.core.controller "gce") }}
   type: NodePort
   {{- end }}
   ports:
-    - name: http
-      port: {{ include "harbor.core.servicePort" . }}
-      targetPort: http
+    - name: {{ ternary "https" "http" .Values.internalTLS.enabled }}
+      port: {{ ternary 443 80 .Values.internalTLS.enabled }}
```

Comment:
Same question here, shouldn't it use `core.containerPorts.*` here instead of hardcoded values?

Reply:
They're not the same values. We use 8080/8443 at container level while we use 80/443 at service level